app/components/article-content.js

```diff
@@ -522,6 +522,11 @@ export default Component.extend(
     },
 
     toogleCollapsibleSection(header) {
+      // Temporary fix for caching issues, it can be removed after all caches expire
+      if (!header.hasAttribute('aria-controls')) {
```

Comment:
`aria-controls` attribute is only added to headers which support collapsing. we don't want to run toggle function for headers which does not support it. 
This code make all header/sections collapsible: https://github.com/Wikia/app/pull/16937 
We just need to wait for cache invalidation. Then we can remove this temp change.